contributing.md

```diff
@@ -114,6 +114,9 @@ The generated changeset file should be included in your commit. That way, when t
 > [!NOTE]
 > If you're not sure what kind of version bump your changes introduce, you can reach out to one of the maintainers in the PR comments and we'll try to help you out!
 
+> [!NOTE]
```

Comment:
Technically, the error that `bun upgrade` fixed was a different error: https://discord.com/channels/1173879003191459860/1175505640156844084/1188685628133625867

`bun i` fixed the error referenced here (but led to the error I linked to, because of out of date bun).

Maybe worth adding "or require of ES module (`ERR_REQUIRE_ESM`)" right before "while trying to commit"?

Reply:
like this?